extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -94,9 +94,15 @@ describe('amp-a4a', () => {
         sandbox.spy(AmpA4A.prototype, 'onAmpCreativeRender');
     getSigningServiceNamesMock.returns(['google']);
     xhrMockJson.withArgs(
-        'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
-        {mode: 'cors', method: 'GET'})
-    .returns(Promise.resolve({keys: [JSON.parse(validCSSAmp.publicKey)]}));
+      'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
+      {
+        mode: 'cors',
+        method: 'GET',
+        requireAmpResponseSourceOrigin: false,
+        disableAmpSourceOrigin: true,
```

Comment:
I was trying to be as explicit as possible thus the word "disable".  Technically this is still a CORS request, it just doesn't include the source origin parameter so ampCors would seem somewhat confusing to me.  Will defer to @cramforce